charts/internal/cloud-provider-config/templates/cloud-provider-config.yaml

```diff
@@ -15,3 +15,6 @@ data:
     local-zone="{{ .Values.zone }}"
     token-url=nil
     node-tags="{{ .Values.nodeTags }}"
+    {{- if .Values.alphaFeatures }}
+    alpha-features="{{range $index, $feature := .Values.alphaFeatures}}{{if $index}},{{end}}{{$feature}}{{end}}"
```

Comment:
Ha, I didn't know that gcp CCM supports such config option. It is confusing and not what I would expect honestly. CCMs have a `--feature-gates` flag and I would expect even out-of-tree provider-specific features to be configured via this flag.